core/src/main/java/org/apache/accumulo/core/clientImpl/NamespaceOperationsHelper.java

```diff
@@ -217,7 +215,7 @@ public void removeConstraint(String namespace, int number)
   public Map<String,Integer> listConstraints(String namespace)
       throws AccumuloException, NamespaceNotFoundException, AccumuloSecurityException {
     Map<String,Integer> constraints = new TreeMap<>();
-    for (Entry<String,String> property : this.getProperties(namespace)) {
+    for (Entry<String,String> property : this.getConfiguration(namespace).entrySet()) {
```

Comment:
Since we're not deprecating, these that are longer/less convenient than the existing code can be reverted back to their former `getProperties`. Sorry for the churn in your PR!